doc/source/starting-ray.rst

```diff
@@ -101,6 +163,35 @@ By default, Ray will parallelize its workload. However, if you need to debug you
 
 Note that some behavior such as setting global process variables may not work as expected.
 
+.. note:: This feature is not supported in Java. But we have the single process mode in Java to help with debugging. See `Single process mode (Java)`_ for details.
+
+Single process mode (Java)
```

Comment:
we can rename Java's single-process mode to local mode too. And combine the Python and Java sections.